test_ai_assistant_response_delay.py

```
@@ -0,0 +1,78 @@
+from datetime import datetime
+
+def get_auth_token():
+    """Get authentication token for API requests"""
+    login_url = "https://cryptouniverse.onrender.com/api/v1/auth/login"
+    login_data = {
+        "email": "test@cryptouniverse.com",
+        "password": "TestPassword123!"
+    }
+    
+    try:
+        login_response = requests.post(login_url, json=login_data)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)